### ISSUE TITLE
bs-platform 7.1.0 not compatible

### ISSUE THREAD

**Issue by swuecho** [darklang/tablecloth#101]
```
 
  Warning number 3 (configured as error) 
  node_modules/tablecloth-bucklescript/src/tablecloth.ml 806:47-63
  
  804 ┆ let capitalize (s : string) : string = String.capitalize s
  805 ┆ 
  806 ┆ let isCapitalized (s : string) : bool = s = String.capitalize s
  807 ┆ 
  808 ┆ let is_capitalized = isCapitalized
  
  deprecated: String.capitalize
Use String.capitalize_ascii instead.
  
  We've found a bug for you!
  node_modules/tablecloth-bucklescript/src/tablecloth.ml
  
  The implementation node_modules/tablecloth-bucklescript/src/tablecloth.ml
       does not match the interface src/tablecloth.cmi:
       ...
       In module Array:
       Values do not match:
         val empty : '_a array
       is not included in
         val empty : 'a array
       File "node_modules/tablecloth-bucklescript/src/tablecloth.mli", line 77, characters 2-22:
         Expected declaration
       File "node_modules/tablecloth-bucklescript/src/tablecloth.ml", line 12, characters 6-11:
         Actual declaration
```

**Comment by ltrls** [comment]
#88 has been merged so all we need is #104 and an updated npm package!

**Comment by pbiggar** [comment]
This is should be fixed now!
